frontend/src/components/pages/chat/components/messageList/MessageList.tsx

```diff
@@ -1,24 +1,22 @@
 import Box from '@material-ui/core/Box';
 import React, { useEffect, useRef, useState } from 'react';
-import { useParams } from 'react-router-dom';
-import { getChatMessages } from '../../../../../services/getChatMessages';
 import { Compose } from '../compose/Compose';
 import { Message } from '../message/Message';
 import './messageList.scss';
 
 const MY_USER_ID = 'apple';
 
 export const MessageList: React.FC = () => {
-  const [messages, setMessages] = useState<Array<any>>([]);
-  const { dialogId } = useParams<any>();
-
-  useEffect(() => {
-    setMessages([]);
-    (async () => {
-      const messages = await getChatMessages(dialogId);
-      setMessages(messages);
-    })();
-  }, [dialogId]);
+  const [messages] = useState<Array<any>>([]);
+  // const { dialogId } = useParams<any>();
+
+  // useEffect(() => {
+  //   setMessages([]);
+  //   (async () => {
+  //     const messages = await getChatMessages(dialogId);
+  //     setMessages(messages);
+  //   })();
+  // }, [dialogId]);
```

Comment:
how we supposed to set messages? 
It is a really bad practice to comment out a code in PR